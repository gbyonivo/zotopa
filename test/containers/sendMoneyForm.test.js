import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { createStore } from 'redux';
import SendMoneyForm from '../../src/containers/sendMoneyForm';
import combineReducers from '../../src/reducers';

const props = {
  send: () => { }
};

const transactReducer = {
  errorSending: null,
  isSending: true
};

describe('SendMoneyForm', () => {
  it('should render correctly', () => {
    const actual = renderer.create(
      <Provider store={createStore(combineReducers({ transactReducer }))}>
        <SendMoneyForm {...props} />
      </Provider>
    );
    expect(actual).toMatchSnapshot();
  });
});