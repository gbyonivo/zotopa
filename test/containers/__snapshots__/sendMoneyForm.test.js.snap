// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendMoneyForm should render correctly 1`] = `
<div
  className="sendMoneyForm"
>
  <h2>
    Send Money
  </h2>
  <div>
    <div
      className="textInput"
    >
      <label
        htmlFor="name"
      >
        Name
      </label>
      <input
        name="name"
        onChange={[Function]}
        placeholder="Ngufan Ivo"
        type="text"
        value=""
      />
    </div>
    <div
      className="textInput"
    >
      <label
        htmlFor="email"
      >
        E-mail
      </label>
      <input
        name="email"
        onChange={[Function]}
        placeholder="orkumaivo@gmail.com"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="numberInput"
  >
    <label
      htmlFor="amount"
    >
      Amount (Â£)
    </label>
    <input
      name="amount"
      onChange={[Function]}
      placeholder="3000"
      type="number"
      value="0"
    />
  </div>
  <div
    className="buttonDiv"
  >
    <button
      disabled={false}
      name="send"
      onClick={[Function]}
    >
      Send Money
    </button>
  </div>
</div>
`;
